perun-core/src/main/java/cz/metacentrum/perun/core/impl/AttributesManagerImpl.java

```diff
@@ -1529,7 +1617,8 @@ public boolean setAttribute(final PerunSession sess, final Object object, final
 			namespace = AttributesManager.NS_ENTITYLESS_ATTR;
 		} else if (object instanceof PerunBean) {
 			PerunBean bean = (PerunBean) object;
-			String name = bean.getBeanName().toLowerCase();
+			// Add underscore between two letters where first is lowercase and second is uppercase, then lowercase BeanName
+			String name = bean.getBeanName().replaceAll("(\\p{Ll})(\\p{Lu})","$1_$2").toLowerCase();
```

Comment:
I'm not sure it will work for RichMember (member attributes). By comment and following code, it converts like``RichMember -> Rich_Member -> rich_member -> _member -> _member_attr_values``. But resulting column should be ``member_attr_values``. Can you please check it or update the comment. Thank you.